modules/RTC/TraceablePeerConnection.js

```diff
@@ -896,7 +898,19 @@ TraceablePeerConnection.prototype._remoteTrackAdded = function(stream, track, tr
         return;
     }
 
-    logger.info(`${this} creating remote track[endpoint=${ownerEndpointId},ssrc=${trackSsrc},type=${mediaType}]`);
+    if (FeatureFlags.isSourceNameSignalingEnabled()) {
+        let sourceName = this.signalingLayer.getTrackSourceName(trackSsrc);
```

Comment:
this needs to be declared outside of the `if` and assigned to `undefined` by default